recipe/meta.yaml

```
@@ -21,7 +21,11 @@ requirements:
 
   run:
     - python >=3.6
+    - cloudpickle >=1.1.1
+    - fsspec >=0.6.0
+    - partd >=0.3.10
     - pyyaml
+    - toolz >=0.8.2
```

It might be worth including `cytoolz` for CPython. Though if we do this, we would need to drop `noarch`. Anyways would be good to hear thoughts on this

```suggestion
    - toolz >=0.8.2
    - cytoolz >=0.8.2  # [python_impl == 'cpython']
```

cc @hmaarrfk

---

I have no opinion on this other than it differs from what we do on pypi, so it might be the source of future confusion.

---

Hmm I'm confused why we would need to drop `noarch` if we add `cytoolz`. From looking at the [dask-feedstock recipe](https://github.com/conda-forge/dask-feedstock/blob/72cc547e826aeb082fb7bc135baf5efb925094db/recipe/meta.yaml) it both has `cytoolz` and is `noarch`

---

cytoolz is not available for pypy.

I guess the conda-forge ecosystem has decided to support pypy.

I think making this a "arch" package is not a problem. especialy with azure.

---

How do things perform when using `cytoolz` with PyPy? Thought there was overhead calling C functions from PyPy

---

We just were not able to compile cytoolz for pypy. There were too many challenges and without support from maintainers, we decided it was not worth it.